buildenv/jenkins/JenkinsfileBase

```diff
@@ -41,7 +41,7 @@ def setupEnv() {
 
 	env.SPEC = "${SPEC}"
 
-	ADOPTOPENJDK_REPO = params.ADOPTOPENJDK_REPO ? params.ADOPTOPENJDK_REPO : "https://github.com/AdoptOpenJDK/openjdk-tests.git"
+	ADOPTOPENJDK_REPO = params.ADOPTOPENJDK_REPO ? params.ADOPTOPENJDK_REPO : "https://github.com/adoptium/aqa-tests.git"
```

Comment:
Can we change `ADOPTOPENJDK_REPO` to `ADOPTIUM_REPO`?

Reply:
Not required (and then would want to update other variable names... and regenerate all test jobs across all Jenkins servers...).

Reply:
Ok, I'll leave the variable names alone for now.